scripts/lighthouseBudget.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-console */
+const fs = require('fs');
+
+const getCategoryScores = data => {
+  const { categories } = data;
+
+  return {
+    ally: categories.accessibility.score * 100,
+    bestPractises: categories['best-practices'].score * 100,
```

Comment:
MIght it be better to use the dot notation here as well for consistency?

Reply:
You can't do a dot notation here because of the `-` character

Reply:
Pardon me. Sorry that escaped me.